primed/dbgap/models.py

```diff
@@ -473,3 +473,11 @@ def get_dbgap_workspaces(self):
             dbgap_consent_code=self.dbgap_consent_code,
         ).order_by("dbgap_version")
         return dbgap_workspaces
+
+    def get_matching_studies(self):
+        """Get the list of studies matching dbGaP study accession phs associated with this data access request."""
+
+        study_list = Study.objects.filter(dbgapstudyaccession__dbgap_phs=self.dbgap_phs).values_list(
+            "short_name", flat=True
+        )
```

Comment:
This should return the list of study objects, not the short names. So take out the values_list method and just return the results from the `filter` call.